types/yanzi/comfort/uplink.js

```diff
@@ -0,0 +1,40 @@
+function consume(event) {
+  let topic = "default";
+
+  event.data.list.forEach((element) => {
```

Comment:
please change this to the format that is used for array uplinks as described by Geppetto

Reply:
@mariusgiger done